<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QMainWindow" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Algorithms on Finite Structures</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QGroupBox" name="closure_groupBox">
      <property name="title">
       <string>CLOSURES</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="nClosure_label">
         <property name="text">
          <string>N</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QGroupBox" name="actionsClosure_groupBox">
         <property name="title">
          <string>Actions</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QScrollArea" name="actions_scrollArea">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="actions_scrollAreaWidgetContent">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>321</width>
               <height>219</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QCheckBox" name="reflex_checkBox">
                <property name="text">
                 <string>Reflexive Closure of R</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="sym_checkBox">
                <property name="text">
                 <string>Symmetric Closure of R</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="transFloyd_checkBox">
                <property name="text">
                 <string>Floyd Transitive Closure of R</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="transDot_checkBox">
                <property name="text">
                 <string>Transitive Closure of R</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="connected_checkBox">
                <property name="text">
                 <string>Connected Components of (Xn , R ∪ R−1)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="subframe_checkBox">
                <property name="text">
                 <string>Subframe of (Xn , R) generated by l =</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="l_spinBox"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QGroupBox" name="RClosure_groupBox">
         <property name="title">
          <string>R</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QScrollArea" name="RClosure_scrollArea">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="R_scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>289</width>
               <height>68</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="Xn_label">
         <property name="text">
          <string>Xn = {}</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QSpinBox" name="nClosure_spinBox"/>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="runClosures_pushButton">
         <property name="text">
          <string>Run</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QGroupBox" name="log_groupBox">
      <property name="title">
       <string>LOG</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QScrollArea" name="log_scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>718</width>
            <height>68</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="modal_groupBox">
      <property name="title">
       <string>MODAL FORMULAS</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="5" column="0">
        <widget class="QLabel" name="nFormula_label">
         <property name="text">
          <string>N</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="5">
        <widget class="QLineEdit" name="formula_lineEdit"/>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="arrow_pushButton">
         <property name="text">
          <string>--&gt;</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="nFormula_spinBox"/>
       </item>
       <item row="11" column="0" colspan="6">
        <widget class="QGroupBox" name="VFormula_groupBox">
         <property name="title">
          <string>V</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QScrollArea" name="VFormula_scrollArea">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents_2">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>349</width>
               <height>52</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="6" column="0" colspan="6">
        <widget class="QGroupBox" name="RFormula_groupBox">
         <property name="title">
          <string>R</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QScrollArea" name="RFormula_scrollArea">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>349</width>
               <height>53</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="5">
        <widget class="QPushButton" name="false_pushButton">
         <property name="text">
          <string>⊥</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QPushButton" name="diamond_pushButton">
         <property name="text">
          <string> ◇ </string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="5">
        <widget class="QLabel" name="A_label">
         <property name="text">
          <string>A =  { ( , ) , ⊥ , → , ♢ ,  p0 , p1 , . . .}.</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="4">
        <widget class="QRadioButton" name="subformulas_radioButton">
         <property name="text">
          <string>Find subformulas of s</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1" colspan="3">
        <widget class="QRadioButton" name="sValid_radioButton">
         <property name="text">
          <string>Is s valid in (Xn , R)</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1" colspan="5">
        <widget class="QRadioButton" name="findX_radioButton">
         <property name="text">
          <string>Find set of points x in Xn such that M, x ⊨ s,
where the valuation of pi in M is Vi.</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="s_label">
         <property name="text">
          <string>Enter s ∈ A ∗</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0" colspan="6">
        <widget class="QPushButton" name="runFormula_pushButton">
         <property name="text">
          <string>Run</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>762</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
